### PR TITLE
Added Lamps+Filing

### PR DESCRIPTION
**Pull request by usmanziak**
Just a general update.

### CODE REVIEW

File: src/edu/ucalgary/ensf409/Filing.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
+package edu.ucalgary.ensf409;
+
+public class Filing {
+
+	
+	
+	private String iD;
```
Comment (mohtashim-khan) [inline src/edu/ucalgary/ensf409/Filing.java]: Filings should extend Furniture, Therefore you dont need the private stirngs for ID, ManuID, type etc. I think if you follow the same format as mackenzie's chair class it should be fine

File: src/edu/ucalgary/ensf409/Filing.java [hunk 2]
```diff
@@ -0,0 +1,80 @@
+package edu.ucalgary.ensf409;
+
+public class Filing {
```
Comment (mohtashim-khan) [inline src/edu/ucalgary/ensf409/Filing.java]: Filing should extend Furniture

File: src/edu/ucalgary/ensf409/Filing.java [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+	private String rails;
+	private String drawers;
+	private String cabinet;
+	private String price;
+	private String manuID;
+	
+	
+	public Filing(String iD, String type, String rails, String drawers, String cabinet, String price, String manuID) {
+		
+		//super(iD, type, rails, drawers, cabinet, price, manuID);
```
Comment (mohtashim-khan) [inline src/edu/ucalgary/ensf409/Filing.java]: Probably use Super here since we want to extend furniture

Comment (mohtashim-khan) [inline src/edu/ucalgary/ensf409/Filing.java]: Probably use Super here since we want to extend furniture

File: src/edu/ucalgary/ensf409/Lamp.java [hunk 4]
```diff
@@ -0,0 +1,65 @@
+package edu.ucalgary.ensf409;
+
+public class Lamp {
```
Comment (mohtashim-khan) [inline src/edu/ucalgary/ensf409/Lamp.java]: Same comments as Filings.java

**Review by mohtashim-khan — CHANGES_REQUESTED** [review]

**Review by mohtashim-khan — APPROVED** [review]
LGTM

**Review by Mackenzie-Chak — APPROVED** [review]
LGTM

**Pull request merged**